docs/supervisor/development.md

```diff
@@ -23,6 +23,72 @@ When the initializing is complete you can access the Home Assistant frontend on
+:::tip
+You must use WSL2. WSL version 1 is not capable of running docker and therefore is incompatible. If using this method, you must ensure you have [set up WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+:::
+
+It is assumed at this point you have installed Debian from the Microsoft Store and [Visual Studio Code](https://code.visualstudio.com/download). From within your WSL2-enabled Debian terminal, run the following annotated bash script, line-by-line.  If you copy the entire script it will perform the required setup of Debian for Home Assitant development.
+
+```bash
+sudo bash << EOF
+set -e #Stop on any error to make it easier to figure out what went wrong
+apt update; apt install -y curl wget git gnupg; #Update and install our dependencies
+curl https://raw.githubusercontent.com/creationix/nvm/master/install.sh | bash; #Install NVM
+source ~/.profile; nvm install node; #reload profile and install node
+sudo apt remove cmdtest yarn  #Remove problematic packages if installed. Debian current Yarn does not work.
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -  #Add yarn repo certificate
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list #Add Yarn repo
+sudo apt update; sudo apt install yarn -y; #Install yarn
+curl -fsSL https://get.docker.com |sudo bash  #install docker Community Edition
+sudo update-alternatives --set iptables /usr/sbin/iptables-legacy #Fix IP Tables for Docker
+git clone https://github.com/home-assistant/supervisor /home/$USER/supervisor #Clone Supervisor to ~/supervisor
+chown $USER:$USER /home/$USER/supervisor
+usermod -aG docker $USER
+systemctl enable docker.service
+service docker start
+sleep 5
+service docker status
+EOF
+
+```
+Restart your Debian terminal to activate the new profiles and software configurations. 
+
+:::tip
+If a problem occurs with an automated script, always run it line by line to figure out where the problem lies.  In this case, skip the first line to directly execute each command.  If you need to completely start over, you can uninstall the Debian app, then from the Windows command prompt execute `del %HOMEPATH%\AppData\Roaming\Code`. After removing your VSCode settings, launch VS Code, and remove all extensions.  Finally reboot your computer and try again. 
+:::
+
+Now you're ready to run Yarn and Node.  We must set up docker communications and launch Visual Studio Code.  The following commands will need to be run after each reboot to link docker to your WSL instance and launch Visual Studio Code. 
+```bash
+sudo mkdir /sys/fs/cgroup/systemd
+sudo mount -t cgroup -o none,name=systemd cgroup /sys/fs/cgroup/systemd
+sudo service docker start
+code
+```
+Visual Studio Code will launch.  If the `code` command does not launch Visual Studio Code, relaunch the Debian terminal, or reboot your computer.
```

Comment:
```suggestion

Visual Studio Code will launch. If the `code` command does not launch Visual Studio Code, relaunch the Debian terminal, or reboot your computer.
```

(PS: Reboot your computer? Windows really hasn't changed in all those years, hasn't it 😁 )